pom.xml

```diff
@@ -114,6 +114,30 @@
       <artifactId>spring-boot-starter-validation</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.springframework.integration</groupId>
+      <artifactId>spring-integration-core</artifactId>
+      <version>6.3.3</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.springframework.integration</groupId>
+      <artifactId>spring-integration-feed</artifactId>
+      <version>5.5.10</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.springframework.integration</groupId>
+      <artifactId>spring-integration-jdbc</artifactId>
+      <version>5.5.10</version> <!-- Замените на нужную версию -->
+    </dependency>
+
+    <dependency>
```

Comment:
Do we need this dependency? (Looks like not)

Reply:
that's correct, we don't need it